File: www/temp.py
import os

with open('locales.js', 'r', encoding='utf-8') as f:
    lines = f.readlines()

new_lines = []
for line in lines:
    new_lines.append(line)
    if 'tellerCap: "קיבולת דלפק"' in line:
        new_lines.append('            tellerFull: "כספר מלא",\n')
    elif 'tellerCap: "Desk Cap"' in line:
        new_lines.append('            tellerFull: "Full Teller",\n')
    elif 'tellerCap: "Cap. Escritorio"' in line:
        new_lines.append('            tellerFull: "Cajero Lleno",\n')

with open('locales.js', 'w', encoding='utf-8') as f:
    f.writelines(new_lines)

File: www/temp.js
const fs = require('fs');

const lines = fs.readFileSync('locales.js', 'utf8').split('\n');
const newLines = [];

for (const line of lines) {
    newLines.push(line);
    if (line.includes('tellerCap: "קיבולת דלפק"')) {
        newLines.push('            tellerFull: "כספר מלא",');
    } else if (line.includes('tellerCap: "Desk Cap"')) {
        newLines.push('            tellerFull: "Full Teller",');
    } else if (line.includes('tellerCap: "Cap. Escritorio"')) {
        newLines.push('            tellerFull: "Cajero Lleno",');
    } else if (line.includes('tellerCap: "Емкость стола"')) {
        newLines.push('            tellerFull: "Полный кассир",');
    }
}

fs.writeFileSync('locales.js', newLines.join('\n'), 'utf8');
